content/_partners/faq._en.md

```diff
@@ -73,59 +69,33 @@ logistics_accordion:
             An Interagency Agreement (IAA) is a type of contract. Login.gov is a cost-recoverable federal service, which means we must, by law, charge other agencies for our work. Our partnership and financial engagement will be governed by the IAA. 
     -
         id: b-2
-        title: What is the timeline for a new IAA?
+        title: Does Login.gov provide authorization?
         content: >-
-            Six to eight weeks. 
+            Login.gov does not provide authorization. At this time, Login.gov supports authentication and identity verification capabilities. We encourage agencies to take the lead on determining the best strategy for their role management and authorization. 
     -
         id: b-3
-        title: What is the timeline for a IAA modifications?
+        title: What is identity verification?
```

Comment:
The [content document](https://docs.google.com/document/d/1NTqpFidS9XBCJ5xFiZ-G2YGb8SilACVLaOeJn7eCkrc/edit#) indicates this should also have "(identity proofing)" in parantheses.